packages/nuxi/src/run.ts

```diff
@@ -2,9 +2,10 @@ import mri from 'mri'
 import type { Command, NuxtCommand } from './commands'
 import { commands } from './commands'
 
-export async function runCommand (command: string, argv = process.argv.slice(2)) {
+export async function runCommand (command: string, argv = process.argv.slice(2), options: any = {}) {
   const args = mri(argv)
   args.clear = false // used by dev
+  args.config = options.config
```

Comment:
This should not be passed from to the `args.config` but instead as second argument of `invoke()`